src/dotnet-uninstall/Shared/Configs/OptionFilterers.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using System.CommandLine;
+using Microsoft.DotNet.Tools.Uninstall.Shared.Filterers;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Shared.Configs
+{
+    internal static class OptionFilterers
```

Comment:
OptionFilterersMap?

Reply:
For the `TODO` in `UninstallCommandExec`, it's indeed not done. The console print is just a placeholder and it should be replaced by the actual uninstallation execution, which is in another PR.

Reply:
[Discussed offline; keep `OptionFilterers`]

Reply:
Replacing `TODO` with `NotImplementedException` is done. I can't reply to and resolve that comment.